docs/training/lerobot_guide.rst

```diff
@@ -359,6 +359,127 @@ Instructions for downloading and using these datasets can be found at the follow
 
+
+
+===============================
+LeRobot X Aloha Solo User Guide
+===============================
+
+The process for using Aloha Solo is similar to Aloha Stationary. However, there are key differences in the configurations, policies, and environment settings that must be applied. This guide outlines the specific changes needed to set up and run Aloha Solo.
+
+Key Differences Between Aloha Solo and Aloha Stationary
+=======================================================
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - **Feature**
+     - **Aloha Stationary**
+     - **Aloha Solo**
+   * - **Robot Configuration**
+     - ``aloha_stationary.yaml``
+     - ``aloha_solo.yaml``
+   * - **Policy**
+     - ``act_aloha_real``
+     - ``act_aloha_solo_real``
+   * - **Environment**
+     - ``aloha_real``
+     - ``aloha_solo_real``
+   * - **Camera Support**
+     - ``cam_high``, ``cam_low``, ``cam_left_wrist``, ``cam_right_wrist``
+     - ``cam_high`` and either ``cam_left_wrist`` or ``cam_right_wrist``
+
+
+Steps to Configure Aloha Solo
+=============================
+
+#. **Place the Required Files**:
+
+   Ensure the following files are updated and placed in their appropriate folder locations:
+
+    - **Robot Configuration**: Place ``aloha_solo.yaml`` in ``lerobot/configs/robot/``. 
+      :download:`Download Robot Configuration <../files/aloha_solo.yaml>`
+
+    - **Policy Configuration**: Place ``act_aloha_solo_real.yaml`` in ``lerobot/configs/policy/``. 
+      :download:`Download Policy Configuration <../files/act_aloha_solo_real.yaml>`
+
+    - **Environment Configuration**: Place ``aloha_solo_real.yaml`` in ``lerobot/configs/env/``. 
+      :download:`Download Environment Configuration <../files/aloha_solo_real.yaml>`
+
+#. **Camera Setup**:
+
+   For Aloha Solo, ensure that only two cameras are configured:
+   - ``cam_high``
+   - Either ``cam_left_wrist`` or ``cam_right_wrist`` (depending on your setup).
+   - 
+#. **Update the Robot Configuration**:
+
+   Replace the robot configuration file used in your setup with ``aloha_solo.yaml``. This file is located in the ``lerobot/configs/robot/`` directory.
+
+   Example:
+
+   .. code-block:: bash
+
+      --robot-path lerobot/configs/robot/aloha_solo.yaml
+
+#. **Adjust the Policy and Environment**:
+
+   Update the policy and environment names in the commands you use:
+     - **Policy**: ``act_aloha_solo_real``
+     - **Environment**: ``aloha_solo_real``
+
+   Example Training Command:
+
+   .. code-block:: bash
+
+      $ DATA_DIR=data python lerobot/scripts/train.py \
+         dataset_repo_id=${HF_USER}/aloha_solo_dataset \
+         policy=act_aloha_solo \
+         env=aloha_solo \
+         hydra.run.dir=outputs/train/act_aloha_solo \
+         hydra.job.name=act_aloha_solo \
+         device=cuda \
+         wandb.enable=false
+
+Example Command Updates for Aloha Solo
+======================================
+
+- **Teleoperation**:
```

Comment:
This is formatting differently than above. I do like this way better though. Align how we do these for stationary and solo

Reply:
Should I keep this? Also, what needs to be done for this 'Align how we do these for stationary and solo'

Reply:
These are exclusive to Aloha Solo. Mobile does not support LeRobot